bokeh/models/mappers.py

```diff
@@ -50,17 +50,47 @@ class LinearColorMapper(ColorMapper):
     this are clamped to ``high``.
     """)
 
-    # TODO: (jc) what is the color code for transparent?
-    # TODO: (bev) better docstring
-    reserve_color = Color("#ffffff", help="""
-    Used by Abstract Rendering.
```

Comment:
Agreed. And although it was added for AR it's potentially a useful feature in general. In any case removing properties should require a deprecation cycle at this point.